### DISCUSSION TITLE
DevOps (Armin)

### DISCUSSION THREAD

**Discussion by arminmor** [WeAreMahsaAmini/Core#234] [Introductions]
Hello, 

My name is Armin and I am a Senior DevOps with over 5 years of experience in the field.

I can help with build and deploy automation, infra as code, architecture and design.

#MahsaAmini
#مهسا_امینی

**Comment by ghost** [comment]
https://github.com/WeAreMahsaAmini/Core/discussions/256

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
I’d appreciate it if you check it here #163

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر
